MVVM Module/Source/Feed UI/Controllers/FeedViewController.swift

```diff
@@ -32,6 +34,13 @@ public final class FeedViewController: UITableViewController, UITableViewDataSou
 				self?.refreshControl?.endRefreshing()
 			}
 		}
+		viewModel?.onErrorStateChange = { [weak self] error in
+			guard let errorMsg = error else {
+				self?.errorView.hideMessage()
+				return
+			}
+			self?.errorView.show(message: errorMsg)
```

Comment:
if/else is more readable in this context:

```suggestion
			if let errorMsg = error {
				self?.errorView.show(message: errorMsg)
			} else {
				self?.errorView.hideMessage()
			}
```